week1/errorMessage.js

```diff
@@ -0,0 +1,4 @@
+export default function errorMessage(response, err) {
+    response.setHeader('Content-Type', 'text/plain');
+    response.write(err);
+}
```

Comment:
You could set your editor to automatically add [new line](https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline) at the end of the file on save!